lib/Target/SubstraitPB/Export.cpp

```diff
@@ -717,8 +717,26 @@ SubstraitExporter::exportOperation(ProjectOp op) {
   return rel;
 }
 
-FailureOr<std::unique_ptr<Rel>>
-SubstraitExporter::exportOperation(UnionDistinctOp op) {
+SetRel::SetOp getSetOp(SetOpKind kind) {
+  switch (kind) {
+  case SetOpKind::unspecified:
+    return ::substrait::proto::SetRel::SET_OP_UNSPECIFIED;
+  case SetOpKind::minus_primary:
+    return ::substrait::proto::SetRel::SET_OP_MINUS_PRIMARY;
+  case SetOpKind::minus_multiset:
+    return ::substrait::proto::SetRel::SET_OP_MINUS_MULTISET;
+  case SetOpKind::intersection_primary:
+    return ::substrait::proto::SetRel::SET_OP_INTERSECTION_PRIMARY;
+  case SetOpKind::intersection_multiset:
+    return ::substrait::proto::SetRel::SET_OP_INTERSECTION_MULTISET;
+  case SetOpKind::union_distinct:
+    return ::substrait::proto::SetRel::SET_OP_UNION_DISTINCT;
+  case SetOpKind::union_all:
+    return ::substrait::proto::SetRel::SET_OP_UNION_ALL;
+  }
```

Comment:
We should try to make this work with a `static_cast`. Since the respective enum values have the same integer correspondance, that should work...